sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -367,6 +372,14 @@ multiFieldRelevanceFunction
         alternateMultiMatchQuery  COMMA alternateMultiMatchField (COMMA relevanceArg)* RR_BRACKET
     ;
 
+altSingleFieldRelevanceFunction
+    : field=relevanceField EQUAL_SYMBOL altSyntaxFunctionName=altSingleFieldRelevanceFunctionName LR_BRACKET query=relevanceQuery RR_BRACKET
```

Comment:
I think we should support arguments too: `(COMMA relevanceArg)*`

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/182/commits/3a88118f8ccd0857050ad71a1e535203ea2c7e2b